src/plugins/sorting-task/SortingTaskNode/ReadView/index.tsx

```diff
@@ -1,23 +1,37 @@
 import React from "react";
 import { ILearningItems } from "../interfaces";
 
+import "./style.scss";
+
 interface IProps {
   learningItems: ILearningItems,
 };
 
 export default class ReadView extends React.PureComponent<IProps> {
 
   public render() {
-    const learningItems = this.props.learningItems.map((learningItem, index) => {
+    const learningItems = this.props.learningItems
+    .filter(learningItem => learningItem.term && learningItem.description)
+    .map((learningItem, index) => {
       return (
         <React.Fragment key={index}>
-          <dt>{learningItem.term}</dt>
-          <dd>{learningItem.description}</dd>
+          <div className="flip-card">
+            <div className="flip-card-inner">
+              <dt className="flip-card-front flip-card-content">
```

Comment:
I wouldn’t recommend to use a definition list in this case. While it might feel a good fit in this case, it might actually cause confusion:

* At any time, there’s only a single element displayed at the same time, i. e. what the user see’s isn’t a real list.
* The term and it’s explanation aren’t displayed at the same time.
* If you try to read out the page’s contents using a screenreader, semantic elements like `dt` and `dd` are announced separately. For the above reasons, it might be confusing.